task/runner.go

```diff
@@ -475,9 +475,17 @@ func errorInfo(err error) *data.ErrorInfo {
 func humanizeCatalystError(err error) error {
 	errMsg := strings.ToLower(err.Error())
 
+	fileNotAccessibleErrs := []string{
+		"504 Gateway Timeout",
+		"giving up after",
```

Comment:
I think we need a little more here, since there are a few places we do HTTP retries and this could happen for any of them

Reply:
Do you think this will be a problem even we filter for both "download error" and "import request" first as is done in [7ae097f](https://github.com/livepeer/task-runner/pull/104/commits/7ae097f62e9b46458f4205f6d81f21bfd4dabde1)?

At the moment, I'm not sure if there is a better way that doesn't involve restructuring catalyst-api error propagation, but I added a TODO and can create an issue to consider restructuring error propagation.

Reply:
You're right, I think that probably is enough - I'd missed the array was being used in combination with those

Reply:
Issue created https://github.com/livepeer/task-runner/issues/105